examples/src/examples/PubEngineSingleUnitExample.java

```diff
@@ -56,8 +53,11 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       // Run Calculation Request
       PubCalculationsApi apiInstance = new PubCalculationsApi(getApiClient());
       ApiResponse<Object> createResponse = apiInstance.postAndCalculateWithHttpInfo(null, null, calcParameters);
-      
-      // Get Calculation Request Status
+      // Comment the above line and uncomment the below lines to add cache control configuration. Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is at max 5 minutes older.
+      // String cacheControlInput = "max-stale=300";
+      // ApiResponse<Object> createResponse = apiInstance.postAndCalculateWithHttpInfo(null, cacheControlInput, calcParameters);
+
+      //Get Calculation Request Status
       ApiResponse<CalculationStatusRoot> getStatus = null;
       File result = null;
       switch (createResponse.getStatusCode()) {
```

Comment:
Can you add `break` in all the case blocks like in other examples?

Reply:
I have added "break" for the PUB Engine single unit example, remaining all other examples are good with 'Switch-Break' statements.